components/popper/src/popper.js

```diff
@@ -32,7 +42,10 @@ const Popper = ({
     const { styles, attributes } = usePopper(referenceElement, popperElement, {
         strategy,
         onFirstUpdate,
-        placement,
+        placement:
+            document.documentElement.dir === 'rtl'
```

Comment:
I don't know what the ideal is here? The [popper](https://popper.js.org/) library that we're using says that it handles RTL, but it seems to provide placement only with physical properties. I assume we want to switch these, so I have done so based on the document.documentElement direction that is being set by app-platform, but I'm not sure if there's a reason why we might want to strictly interpret the placement as a physical property?